usaspending_api/reporting/v2/views/agencies/overview.py

```diff
@@ -148,8 +152,10 @@ def format_results(self, result_list):
                     "missing_tas_accounts_count": result["missing_tas_accounts_count"],
                 },
                 "obligation_difference": result["obligation_difference"],
-                "unlinked_contract_award_count": 0,
-                "unlinked_assistance_award_count": 0,
+                "unlinked_contract_award_count": result["unlinked_procurement_c_awards"]
+                + result["unlinked_procurement_d_awards"],
+                "unlinked_assistance_award_count": result["unlinked_assistance_c_awards"]
+                + result["unlinked_assistance_d_awards"],
```

Comment:
This will not work since the sorting is performed in the database, so the returned values will need to be calculated in the DB

Reply:
> 
> 
> I don't think sorting is functional

Correct, I've updated and new test passes @tony-sappe